atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -512,17 +512,22 @@ def get_build(self, metadata):
         if koji_task_id is not None:
             self.log.info("build configuration created by Koji Task ID %s",
```

Comment:
This is very minor, but it might be a good idea to use %r, instead of %s. The side effect is that this prints the representation of `koji_task_id` instead of its value:

```
In [22]: logger.warning('Task ID %r', '1234')
WARNING:tcpserver:Task ID '1234'

In [23]: logger.warning('Task ID %s', '1234')
WARNING:tcpserver:Task ID 1234
```